src/transformers/modeling_utils.py

```diff
@@ -2629,7 +2629,11 @@ def _fix_key(key):
         # This is not ideal in terms of memory, but if we don't do that not, we can't initialize them in the next step
         if low_cpu_mem_usage:
             for key in missing_keys:
-                if key.startswith(prefix):
+                if key in list(model_state_dict.keys()):
+                    key = key
+                elif ".".join([prefix] + key.split(".")) in list(model_state_dict.keys()):
```

Comment:
This is just `f"{prefix}.key"` I believe

Reply:
In Line 2634, yes is just f"{prefix}.key"

Reply:
So let's use it please.